playbooks/demo_hw.yml

```diff
@@ -0,0 +1,83 @@
+# Manually verify your hosts has FQDN set correctly
+# check it's in /etc/hosts /etc/hostname and hostname returns it, /etc/hosts first line should be 192.168.73.1  $fqdn
```

Comment:
Our etc_hosts role do this. If not, you should ensure the ansible inventory has the correct IP.

Reply:
I'm not sure if this is the way how I'm running it, but it didn't work for systems that couldn't resolve their FQDN. I ran ansible-playbook with `-i machine.example.com,` where machine.example.com was on my localhost /etc/hosts. I think this comment does not do any harm and checking hostname is properly is good idea anyway.